locales/zh_CN.json

```diff
@@ -0,0 +1,1116 @@
+    },
+    "color-contrast-enhanced": {
+      "description": "确保前景色与背景色之间的对比度满足 WCAG 2 AAA 增强对比度比率阈值",
+      "help": "元素必须满足增强的颜色对比度比率阈值"
+    },
+    "color-contrast": {
+      "description": "确保前景色与背景色之间的对比度满足 WCAG 2 AA 最小对比度比率阈值",
+      "help": "元素必须满足最小颜色对比度比率阈值"
+    },
+    "css-orientation-lock": {
+      "description": "确保内容不锁定于任何特定显示方向，并且在所有显示方向下都可操作",
+      "help": "CSS 媒体查询不得锁定显示方向"
+    },
+    "definition-list": {
+      "description": "确保 <dl> 元素结构正确",
+      "help": "<dl> 元素必须只直接包含按正确顺序排列的 <dt> 和 <dd> 组、<script>、<template> 或 <div> 元素"
+    },
+    "dlitem": {
+      "description": "确保 <dt> 和 <dd> 元素被 <dl> 包含",
+      "help": "<dt> 和 <dd> 元素必须被 <dl> 包含"
+    },
+    "document-title": {
+      "description": "确保每个 HTML 页面包含一个非空的 <title> 元素",
+      "help": "页面必须有 <title> 元素以帮助导航"
+    },
+    "duplicate-id-active": {
+      "description": "确保激活元素的每个 id 属性值是唯一的",
+      "help": "激活元素的 ID 必须唯一"
+    },
+    "duplicate-id-aria": {
+      "description": "确保在 ARIA 和标签中使用的每个 id 属性值是唯一的",
+      "help": "在 ARIA 和标签中使用的 ID 必须唯一"
+    },
+    "duplicate-id": {
+      "description": "确保每个 id 属性值是唯一的",
+      "help": "id 属性值必须是唯一的"
+    },
+    "empty-heading": {
+      "description": "确保标题有可辨识的文本",
+      "help": "标题不应该为空"
+    },
+    "empty-table-header": {
+      "description": "确保表格标题有可辨识的文本",
+      "help": "表格标题文本不应该为空"
+    },
+    "focus-order-semantics": {
+      "description": "确保焦点顺序中的元素有适合交互内容的角色",
+      "help": "焦点顺序中的元素应该有适当的角色"
+    },
+    "form-field-multiple-labels": {
+      "description": "确保表单字段没有多个标签元素",
+      "help": "表单字段不得有多个标签元素"
+    },
+    "frame-focusable-content": {
+      "description": "确保有可聚焦内容的 <frame> 和 <iframe> 元素没有设置 tabindex=-1",
+      "help": "有可聚焦内容的框架不得设置 tabindex=-1"
+    },
+    "frame-tested": {
+      "description": "确保 <iframe> 和 <frame> 元素包含 axe-core 脚本",
+      "help": "框架应该使用 axe-core 进行测试"
+    },
+    "frame-title-unique": {
+      "description": "确保 <iframe> 和 <frame> 元素包含唯一的 title 属性",
+      "help": "框架必须有一个唯一的 title 属性"
+    },
+    "frame-title": {
+      "description": "确保 <iframe> 和 <frame> 元素有可访问名称",
+      "help": "框架必须有可访问名称"
+    },
+    "heading-order": {
+      "description": "确保标题的顺序在语义上是正确的",
+      "help": "标题级别只应递增一"
+    },
+    "hidden-content": {
+      "description": "向用户通报隐藏内容",
+      "help": "页面上的隐藏内容应该被分析"
+    },
+    "html-has-lang": {
+      "description": "确保每个 HTML 页面有 lang 属性",
+      "help": "<html> 元素必须有一个 lang 属性"
+    },
+    "html-lang-valid": {
+      "description": "确保 <html> 元素的 lang 属性包含有效值",
+      "help": "<html> 元素的 lang 属性必须有一个有效值"
+    },
+    "html-xml-lang-mismatch": {
+      "description": "确保包含有效的 lang 和 xml:lang 属性的 HTML 元素在页面的基础语言上一致",
+      "help": "有 lang 和 xml:lang 属性的 HTML 元素必须有相同的基础语言"
+    },
+    "identical-links-same-purpose": {
+      "description": "确保有相同可访问名称的链接服务于类似的目的",
+      "help": "有相同名称的链接必须有相似的目的"
+    },
+    "image-alt": {
+      "description": "确保 <img> 元素有替代文本或 role 属性为 \"none\" 或 \"presentation\"",
```

Comment:
"确保 <img> 元素有替代文本或角色属性为 \"none\" 或 \"presentation\""

Reply:
Why is "角色" used here instead of the "role" mentioned in other comments?

Reply:
I think it would be better to directly use the English term "role" here. Therefore, the role mentioned here refers to a specific attribute name within an element, such as the "id" attribute, which may be more difficult to understand when translated into Chinese.

Reply:
I think it would be better to directly use the English term "role" here. Therefore, the role mentioned here refers to a specific attribute name within an element, such as the "id" attribute, which may be more difficult to understand when translated into Chinese.

Reply:
That's fair.